DS13/code/modules/mob/living/carbon/human/species_types/t800.dm

```diff
@@ -0,0 +1,225 @@
+/datum/species/t800 //Badmin robot of death.
+	name = "T-800"
+	id = "t800"
+	say_mod = "states"
+	sexes = FALSE
+	species_traits = list(NOBLOOD,NO_UNDERWEAR,NOEYES)
+	inherent_traits = list(TRAIT_VIRUSIMMUNE,TRAIT_NODISMEMBER,TRAIT_NOLIMBDISABLE,TRAIT_NOHUNGER,TRAIT_RESISTHEAT,TRAIT_NOBREATH,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_NOFIRE)
+	inherent_biotypes = list(MOB_ROBOTIC, MOB_HUMANOID)
+	meat = null
+	damage_overlay_type = "synth"
+	limbs_id = "t800"
+	armor = 80 //Considering they can just transporter it off...
+	punchdamagelow = 10
+	punchdamagehigh = 19
+	punchstunthreshold = 14 //about 50% chance to stun
+	no_equip = list(SLOT_WEAR_MASK, SLOT_WEAR_SUIT, SLOT_GLOVES, SLOT_SHOES, SLOT_W_UNIFORM, SLOT_S_STORE)
+	nojumpsuit = TRUE
+	stunmod = 0 //Can't be stunned
+	speedmod = 3 //pretty fucking slow
+	attack_sound = 'sound/weapons/resonator_blast.ogg'
+	miss_sound = 'sound/weapons/parry.ogg'
+
+
+/datum/species/t800/spec_attacked_by(obj/item/I, mob/living/user, obj/item/bodypart/affecting, intent, mob/living/carbon/human/H)
+	if(istype(I, /obj/item/weldingtool))
```

Comment:
Have it check for if its attacked on help intent to have it repair. Otherwise people attacking it with a welder will instead fix it.